Classes/DataHandling/Operation/AbstractRecordOperation.php

```diff
@@ -313,6 +311,10 @@ private function resolveStoragePid(): int
             $settings['persistence.']['storagePid.'] ?? []
         );
 
+        if ($pid === '') {
```

Comment:
Should this also cover the null check not only empty string?

Reply:
Exchanging this condition with `empty($pid)`.